modal-go/test/sandbox_test.go

```diff
@@ -725,3 +725,39 @@ func TestSandboxInvalidTimeouts(t *testing.T) {
 	g.Expect(err).Should(gomega.HaveOccurred())
 	g.Expect(err.Error()).Should(gomega.ContainSubstring("whole number of seconds"))
 }
+
+func TestSandboxExperimentalDocker(t *testing.T) {
```

Comment:
Mentioned on Slack, but would it be sufficient to just intercept the request and assert that the right proto field is set?